app/src/main/java/campus/tech/kakao/map/view/SearchPlaceActivity.kt

```diff
@@ -32,31 +36,46 @@ class SearchPlaceActivity : AppCompatActivity() {
         //Place 리사이클러뷰 설정
         val recyclerView = binding.recyclerView
         recyclerView.layoutManager = LinearLayoutManager(this)
-        placeAdapter = viewModel.vmPlaceAdapter
+
+        placeAdapter = PlaceAdapter(listOf()) { place ->  //리사이클러뷰의 아이템을 누르면
+
+            viewModel.insertSavedSearch(place)
+            viewModel.updateSavedSearch()
+            viewModel.setSharedPreferences(place)
+            viewModel.itemClick(place)
+
+        }
+
         binding.recyclerView.adapter = placeAdapter
 
         //savedSearch 저장된 검색어 설정
         val savedSearch = binding.savedSearch
         savedSearch.layoutManager = LinearLayoutManager(this, LinearLayoutManager.HORIZONTAL, false)
-        savedSearchAdapter = viewModel.vmSavedSearchAdapter
-        binding.savedSearch.adapter = savedSearchAdapter
 
-        viewModel.updateSavedSearch()
+        savedSearchAdapter = SavedSearchAdapter(listOf(),
+            onCloseClick = { SavedSearch -> //SavedSearch의 x를 누르면
+                viewModel.deleteSavedSearch(SavedSearch.id)
+                viewModel.updateSavedSearch()
+            },
+            onNameClick = { SavedSearch ->   //SavedSearch의 이름을 누르면
+                viewModel.nameClick(SavedSearch)
+                viewModel.setSearchText(SavedSearch)
+            }
+        )
 
+        binding.savedSearch.adapter = savedSearchAdapter
 
+        viewModel.updateSavedSearch()
 
 
         //-----viewModel observe-----------------------------------------
         val activity = this
         with(viewModel) {
 
-            //PlaceAdapter
+            //PlaceAdapter -->Parcelable 이용해보기
             itemClick.observe(activity, Observer { place ->  //Place 클릭 했을 때
                 val resultIntent = Intent()
-                resultIntent.putExtra("name", place.name)
-                resultIntent.putExtra("address", place.address)
-                resultIntent.putExtra("latitude", place.latitude)
-                resultIntent.putExtra("longitude", place.longitude)
+                resultIntent.putExtra("location", place.toLocation())
```

Comment:
👍 
Intent에 데이터 담을 때 쓰는 "location"와 같은 키값을 아래와 같이 상수로 선언해서 직접 하드코딩하지 않도록 해주시면 좋겠습니다!
```suggestion
                resultIntent.putExtra(INTENT_EXTRA_LOCATION, place.toLocation())
                
                companion object{
                    const val INTENT_EXTRA_LOCATION = "location"
                }
```